src/HardwareNFCConnection.cpp

```
@@ -107,67 +124,71 @@ bool HardwareNFCConnection::begin() {
     // BUSY signal indicates RF subsystem boot state; timeout detects dead/unreliable chips
     unsigned long start = millis();
     while (digitalRead(pinBusy_) == HIGH) {
         if (millis() - start > 2000) {
-            Serial.println("HardwareNFCConnection: TIMEOUT waiting for BUSY LOW after reset!");
+            logBuf.logPrintf("HardwareNFCConnection: TIMEOUT waiting for BUSY LOW after reset!\n");
             break;
         }
         delay(1);
     }
-    Serial.printf("HardwareNFCConnection: BUSY went LOW after %lums\n", millis() - start);
+    logBuf.logPrintf("HardwareNFCConnection: BUSY went LOW after %lums\n", millis() - start);
```

Clang (14.0.6)

[warning] do not call c-style vararg functions

(cppcoreguidelines-pro-type-vararg)